File: Programação Para Dispositivos Móveis/Exercícios em Sala App/Aula 04/Console Antigo .java/ConsolesAntigos/app/src/main/java/com/pedroferreira/consolesantigos/ConsolesAntigos.java
package com.pedroferreira.consolesantigos;

public class ConsolesAntigos {
    private String nome;
    private int anoLancamento;
    private String publicadora;
    private String historia;

    public ConsolesAntigos(String nome, int anoLancamento, String publicadora, String historia) {
        this.nome = nome;
        this.anoLancamento = anoLancamento;
        this.publicadora = publicadora;
        this.historia = historia;
    }

    public String getNome() {
        return nome;
    }

    public int getAnoLancamento() {
        return anoLancamento;
    }

    public String getPublicadora() {
        return publicadora;
    }

    public String getHistoria() {
        return historia;
    }


}

File: Programação Para Dispositivos Móveis/Exercícios em Sala App/Aula 05/Console Antigo em .kt/app/src/main/java/com/example/consolesantigos/ConsolesAntigos.kt
package com.example.consolesantigos

class ConsolesAntigos(
    private val nome: String,
    private val anoLancamento: Int,
    private val publicadora: String,
    private val historia: String
) {
    fun getNome(): String {
        return nome
    }

    fun getAnoLancamento(): Int {
        return anoLancamento
    }

    fun getPublicadora(): String {
        return publicadora
    }

    fun getHistoria(): String {
        return historia
    }
}